ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1699,3 +1699,31 @@ FCU_DSP_DFVSC_DTM_RTM:
   - DTM
   - RTM
 
+
+FCU_DXZTC_DFSS:
+  guid: "95515cb5-eb97-4acf-9be2-a792c8cf99e4"
+  description: "FCU with SS, discharge temp sensors, zone control and sensor, filter change"
+  is_canonical: true
+  implements:
+  - FCU
+  - DXZTC
+  - DFSS
+  opt_uses:
+  - failed_alarm
+  - filter_alarm
+
+FCU_RHDHS_DXDC_DFSS:
+  guid: "6d3bf040-cb9a-4ce9-8c3e-30607fb8977d"
+  description: "FCU with temperature & pressure monitor, humidity monitor and water leak & status alarm."
+  is_canonical: true
+  implements:
+  - FCU
+  - RHDHS
+  - DXDC
+  - DFSS
+  opt_uses:
+  - water_leak_detection_alarm
```

Comment:
Please clarify what water leaks you are detecting? This FCU has compressors so are you trying to monitor leaks from the condensate pan?

Reply:
yes correct, ma'am. Is there existing field you know like that I can use?

Reply:
please use `water_leak_alarm` as this is the field being used on most FCU types.

Reply:
i see, noted on this. Thanks